tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -592,6 +592,21 @@ Then(
   }
 )
 
+Then(
+  'file/folder/resource {string} should not be listed in the folder {string} on the webUI',
+  async function(file, target) {
+    await client.page.filesPage().navigateAndWaitTillLoaded(target)
+    return client.page.FilesPageElement.filesList()
+      .isElementListed(file, 'file', client.globals.waitForNegativeConditionTimeout)
+      .then(state => {
+        const message = state
+          ? `Error: Folder '${file}' is listed on the '${target}'`
+          : `File '${file}' is not listed on the '${target}'`
+        return client.assert.ok(!state, message)
+      })
```

Comment:
please simplify this, we dont need to have a error message `File '${file}' is not listed on the '${target}'`
we just need error message for when the resource is listed on the webUI

Reply:
done, @dpakach Please review again in the PR https://github.com/owncloud/web/pull/4613